app/src/androidTest/java/mozilla/lockbox/OnboardingTest.kt

```diff
@@ -34,4 +37,33 @@ open class OnboardingTest {
         }
         navigator.checkAtItemList()
     }
+
+    @Test
+    fun autofillSkipButtonNavigatesToItemList() {
+        navigator.gotoAutofillOnboarding()
+        autofillOnboardingScreen { tapSkip() }
+        navigator.checkAtOnboardingConfirmation()
+    }
+
+    @Test
+    fun autofillGoToSettingsNavigatesToSystemSettings() {
+        navigator.gotoAutofillOnboarding()
+        autofillOnboardingScreen {
+            exists()
+            touchGoToSettings()
+        }
+    }
+
+    @Test
+    fun fxaLoginToOnboardingToItemList() {
+        navigator.gotoFxALogin()
+        fxaLogin { tapPlaceholderLogin() }
+        navigator.checkAtFingerprintOnboarding()
```

Comment:
if this code is run on a device without fingerprint capabilities, it will fail (I think) :(

Reply:
changed this to match the navigator logic to check for fingerprint + autofill capabilities